lib/individual/health.dart

```diff
@@ -1,6 +1,9 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
I don't think there is any need for importing cupertino.dart file as it is already included in the material.dart import. Please remove it :thought_balloon: